protocols/upnp/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "libp2p-upnp"
+edition = "2021"
+rust-version = "1.60.0"
+description = "UPnP support for libp2p transports"
+version = "0.1.0"
+license = "MIT"
+repository = "https://github.com/libp2p/rust-libp2p"
+keywords = ["peer-to-peer", "libp2p", "networking"]
+categories = ["network-programming", "asynchronous"]
+
```

Comment:
You will probably need `publish = false` temporarily to make CI happy. See https://github.com/obi1kenobi/cargo-semver-checks-action/issues/53.

Reply:
ok, thanks updated